style.css

```diff
@@ -95,3 +53,50 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+body {
+    font-family: Arial, sans-serif;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+}
+
+.tooltip {
+    display: none;
+    position: fixed;
+    background-color: #333;
+    color: #fff;
+    padding: 5px 10px;
+    border-radius: 8px;
+    z-index: 1;
```

Comment:
it is better to show tooltips over modals, because it is possible to have something with tooltips inside modals.
Just as note